maven-extension/src/main/java/io/opentelemetry/maven/SpanRegistry.java

```diff
@@ -134,7 +148,9 @@ public static MavenProjectKey fromMavenProject(@Nonnull MavenProject mavenProjec
 
     abstract String pluginArtifactId();
 
-    static MojoExecutionKey fromMojoExecution(MojoExecution mojoExecution) {
+    abstract MavenProjectKey projectKey();
```

Comment:
Where is `projectKey()` used?

Reply:
It is used by the Map of spans by `mojoExecutionKey` of the SpanRegistry. It's needed when using Maven Parallel Builds as the same mojo goal can be executed simultaneously on multiple Maven sub projects.

See https://github.com/open-telemetry/opentelemetry-java-contrib/blob/v1.9.0/maven-extension/src/main/java/io/opentelemetry/maven/SpanRegistry.java#L23